private-templates-service/client/src/components/Common/ShareModal/ShareModalForm/ShareModalForm.tsx

```diff
@@ -31,23 +40,9 @@ const mapDispatchToProps = (dispatch: any) => {
   }
 }
 
-class ShareModalForm extends React.Component<ShareModalFormProps, State> {
-
-  constructor(props: ShareModalFormProps) {
-    super(props);
-    this.state = { value: '' };
-
-    this.handleChange = this.handleChange.bind(this);
-    this.handleSubmit = this.handleSubmit.bind(this);
-  }
-
-  handleChange(event: any) {
-    this.setState({ value: event.target.value });
-  }
+class ShareModalForm extends React.Component<ShareModalFormProps> {
 
   handleSubmit(event: any) {
```

Comment:
how does this work without `this.handleSubmit.bind(this);`

can you change it into `handleSubmit = () => { ... }`

Also, since you are no longer using a form there is no need for `event.preventDefault()` so you can get rid of `event` in the params

Reply:
addressed